### PR TITLE
Milestone 1: Adding features to master branch

### PR DESCRIPTION
**Pull request by SaidRasinlic**
In this module I did the following things

- added html and css file
- added .gitignore that ignores "test.md" file
- added "linters.yml" file for html-css
- used README.md Microverse template

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+"This file should be ignored by git".
```
Comment (DammyShittu) [inline test.md:1]: - Kindly add this file to the `.gitignore` file as the requirement says.

**Review by DammyShittu — COMMENTED** [review]
Hi @SaidRasinlic , 

Great job so far!

There are some issues that you still need to work on to go to the next project but you are ALMOST THERE!


# Required Changes ♻️

- It would be great to have a very good PR description. PR descriptions serve as summaries for the changes or features that were worked on in that particular PR. Please, check this [article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects) to see how to write descriptive PR descriptions. 😀

- For some reason, I am unable to select either `Approve` or `Request Changes`. Kindly check the settings of your repository to get this fixed. 👍  Attached is a screenshot of what is being shown


_Check the other comment under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SaidRasinlic** [comment]
The reason why You couldn't approve PR is that because I limited reviews on all repositories (My Profile>Settings>Code review limits). I successfully changed it so there shouldn't be any problems.

File: test.md (hunk 1 above)
Comment (SaidRasinlic) [reply test.md:1]: added

**Comment by SaidRasinlic** [comment]
The reason why You couldn't approve PR is that because I limited reviews on all repositories (My Profile>Settings>Code review limits). I successfully changed it so there shouldn't be any problems.

**Comment by DammyShittu** [comment]
> The reason why You couldn't approve PR is that because I limited reviews on all repositories (My Profile>Settings>Code review limits). I successfully changed it so there shouldn't be any problems.

Oh.
Alright. 
That's fine.

Happy coding 🚀

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi @SaidRasinlic ,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍 
- Project is set up correctly ✔️ 
- Descriptive Pull Request ✔️ 
- No linter errors ✔️ 
- Professional documentation ✔️ 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**